core/src/modules/configLoaders/index.ts

```diff
@@ -150,6 +180,21 @@ export async function processConfigObjects(
     if (Object.keys(configObject).length === 0) continue;
     let klass = configObject?.class?.toLowerCase();
     let ids: IdsByClass;
+
+    const externallyValidate = await shouldExternallyValidate(
+      canExternallyValidate,
+      configObject,
+      configObjects,
+      locallyValidateIds
+    );
+
+    if (!externallyValidate) {
+      log(
+        `[ config ] skipping external validation for ${configObject.class} \`${configObject.id}\``,
+        "notice"
+      );
+    }
```

Comment:
This message is interesting! If I `grouparoo validate --local data_warehouse` (assuming data_warehouse is an App), I think I would see this message a lot of times - for the sources, schedules, and properties.   I guess that's OK for now, as it's certainly clear, but perhaps we only want to show it for the parent objects?

Reply:
We've used it a few times in some of the plugins, but yeah I dont think it's been used in core

Reply:
Would collecting everything that's being locally validated and logging it in a single message at the end be better? That way there arent a bunch of these in between the validation messages?

Reply:
I think the way you have it is OK for now!   Let's get this merged in and see how annoying it gets